app.py

```diff
@@ -28,15 +37,29 @@ class ReuploadDocbins(BaseModel):
 @app.post("/tokenize_record")
 def tokenize_record(request: Request) -> Tuple[int, str]:
     session_token = general.get_ctx_token()
-    value = util.tokenize_record(request.project_id, request.record_id)
+    value = tokenization_manager.manage_record_tokenization(
+        request.project_id, request.record_id
+    )
+    general.remove_and_refresh_session(session_token)
+    return value, ""
+
+
+@app.post("/tokenize_calculated_attribute")
+def tokenize_record(request: AttributeTokenizationRequest) -> Tuple[int, str]:
+    session_token = general.get_ctx_token()
+    value = task_manager.start_tokenization_task(
```

Comment:
in an early iteration this was designed meant to return a different integer if something went wrong. This isn't the case we  pretty much always return 200 so we don't need to store the value imo

Reply:
Removed value variable and now returns always 200